.buildkite/al2_test.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+set -eu
+
+source ./.buildkite/al2env.sh
+
+export PATH=$bin_path:$PATH
+
+sudo -E "FIRECRACKER_CONTAINERD_RUNTIME_CONFIG_PATH=$runtime_config_path" $bin_path/firecracker-containerd --config $dir/config.toml &
+containerd_pid=$!
+sudo $bin_path/firecracker-ctr --address $dir/containerd.sock content fetch docker.io/library/alpine:3.10.1
+sudo -E env "PATH=$PATH" /usr/local/bin/go test -run TestMultipleVMs_Isolated ./...
+
+sudo kill -9 $containerd_pid
```

Comment:
That's certainly...forceful.  It would be nicer to send `SIGTERM` and let the process gracefully exit, but then you need to monitor it and make sure it actually does.  @nmeyerhans, do you have an opinion here?

Reply:
I've wen't ahead and removed the `SIGKILL`.